tests/fixtures/expected/blocks/directives/sidebar.html

```diff
@@ -1,5 +1,5 @@
-<div class="admonition-wrapper"><div class="admonition admonition-sidebar"><p class="sidebar-title"><span>The sidebar's title</span></p><p>some text inside sidebar</p>
-</div></div>
+<div class="admonition admonition-sidebar "><p class="sidebar-title"><span>The sidebar's title</span></p><p>some text inside sidebar</p>
+</div>
```

Comment:
I've verified this change using the web inspector on symfony.com. It doesn't change anything in the rendering and styles.

Reply:
Thanks for checking. You are right because it looks like sidebar was the only admonition that still added the `admonition-wrapper`:

![admonitions](https://user-images.githubusercontent.com/73419/224030742-303740e4-ad5e-4c2b-9c69-33bde48a1a50.png)